Source/Scene/ImageryLayer.js

```diff
@@ -955,30 +945,16 @@ define([
 
         reproject.vertexArray.getAttribute(1).vertexBuffer.copyFromArrayView(webMercatorT);
 
-        var command = new ClearCommand({
-            color : Color.BLACK,
-            framebuffer : reproject.framebuffer
-        });
-        command.execute(context);
-
-        if ((!defined(reproject.renderState)) ||
-                (reproject.renderState.viewport.width !== width) ||
-                (reproject.renderState.viewport.height !== height)) {
-
-            reproject.renderState = RenderState.fromCache({
-                viewport : new BoundingRectangle(0, 0, width, height)
-            });
-        }
-
-        var drawCommand = new DrawCommand({
-            framebuffer : reproject.framebuffer,
+        var computeCommand = new ComputeCommand({
             shaderProgram : reproject.shaderProgram,
-            renderState : reproject.renderState,
-            primitiveType : PrimitiveType.TRIANGLES,
+            outputTexture : outputTexture,
+            uniformMap : uniformMap,
             vertexArray : reproject.vertexArray,
-            uniformMap : uniformMap
+            persists : true,
+            owner : imageryLayer
         });
-        drawCommand.execute(context);
+
+        computeCommand.execute(context);
```

Comment:
Recall that we only want the `Scene` to execute commands so we should add this compute command to the command list.